cfe_internal/core/watchdog/templates/watchdog.mustache

```diff
@@ -1,8 +1,152 @@
 #!/usr/bin/sh
 
-# If cf-execd is present and executable, and there is no cf-execd process running, then launch it.
+set -e
 
-[ -x {{{vars.sys.cf_execd}}} ] && if ! ( ps -ef | grep -q {{{vars.sys.bindir}}}/[c]f-execd ); then
-  {{{vars.sys.cf_execd}}}
+# Watchdog events are logged here.
+LOGFILE="/tmp/watchdog.log"
+touch ${LOGFILE}
+TMPDIR="/tmp"
+CFENGINE_WORKDIR="/tmp/var/cfengine"
+CFENGINE_WORKDIR_COLLECTION=""
```

Comment:
Do you need to `mkdir -p "${CFENGINE_WORKDIR}"`?

Reply:
Well, I wasn't going to, there is a bit of that sprinkeled in just because of how i was testing on a buildslave without taking it offline.